app/scrapers/generalized.py

```diff
@@ -42,8 +42,12 @@ def get_page(self, query, startIndex=0, qtype=''):
         if self.name == 'mojeek' and qtype == 'news':
             payload['fmt'] = 'news'
         response = requests.get(url, headers=self.headers, params=payload)
+        status_code = response.status_code
+        print(status_code)
```

Comment:
Remove this statement. IMO, we can use logging to give detail logs on console. Let me know your opinion

Reply:
@vaibhavsingh97 I don't know about this `logging`.(I thought print is only used to debug) Can you please point me to a link to read on this.

Reply:
You can look [here](https://docs.python.org/2/library/logging.html) and [here](http://flask.pocoo.org/docs/dev/logging/)